Packs/MicrosoftEntraID/README.md

```diff
@@ -1,77 +1,60 @@
 <~XSIAM>
 
-### This pack includes:
-- Log Normalization - XDM mapping for the following datasets:
-  - msft_azure_raw
-  - msft_azure_ad_raw
-  - msft_azure_ad_audit_raw
-
-### Supported Event Types:
-- AuditLogs
-- SignInLogs
-- NonInteractiveUserSignInLogs
-- ServicePrincipalSignInLogs
-- ManagedIdentitySignInLogs
-- ADFSSignInLogs
-- ProvisioningLogs
-- RiskyUsers
-- UserRiskEvents
-- RiskyServicePrincipals
-- ServicePrincipalRiskEvents
-
-### Supported Timestamp Formats:
-* For *msft_azure_ad_raw*, timestamp ingestion is according to the **createdDateTime** field in UTC (00:00) time zone.
-* For *msft_azure_ad_audit_raw*, timestamp ingestion is according to the **ActivityDateTime** field in UTC (00:00) time zone.
-* For *msft_azure_raw*, depending on the **category** timestamp, ingestion is according to the **createdDateTime** and **ActivityDateTime** fields in UTC (00:00) time zone.
+## What does this pack do
 
-***
+The Azure Logs pack contains data normalization rules for ingesting various Azure log types that are collected via the [Office 365](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-from-Microsoft-Office-365) and [Azure Event Hub](https://docs-cortex.paloaltonetworks.com/r/Cortex-XSIAM/Cortex-XSIAM-Documentation/Ingest-logs-from-Microsoft-Azure-Event-Hub) data sources, and modeling them to the Cortex Data Model (XDM).
```

Comment:
It was originally just for Entra ID logs, but recently we were requested to rename it to "Azure Logs" so that it could  be used as a more general pack for Azure.
We couldn't change the originating directory name, so it is reflected only externally on the pack_metadata.json display name entry.